dotcom-rendering/docs/interactives/working-with-interactives.md

```diff
@@ -0,0 +1,34 @@
+# Working with Interactives
+
+Interactive is a Design type for rich content pieces - typically with custom
+Javascript for interactive effects.
+
+DCR aims to support interactive development in a variety of ways.
+
+_Note, this document is a work in progress and more will be added shortly._
+
+## Templating
+
+Interactives support placeholders for key article metadata. This is particularly
+useful for immersive interactives, which do not come with a standard top-meta
+section, but they work for any interactive page.
+
+To use, simply add a variable to your interactive markup and DCR will populate
+it when it builds the page. A moustache-like syntax is supported:
+
+    <div>Some content with a {{ byline }} in it.</div>
+
+Unsupported variables will be left as-is (including the curlies).
+
+Supported variables are:
```

Comment:
Might be helpful here to link to an example article and quote the example elements present.

Eg: 
```
Example:
    byline: "Jamie Grierson"
    sectionTag: "Coronavirus"
    standfirst: "Neil Ferguson says immunity levels falling behind other countries that have jabbed 12- to 15-year-olds"
    trailText: (not sure what this is)
    webPublicationDate: "Mon 13 Sep 2021 08.53 BST"
    webTitle: "Vaccinating teenagers against Covid is priority, says UK epidemiologist"
    webURL: "https://www.theguardian.com/world/2021/sep/13/vaccinating-teenagers-against-covid-is-priority-says-uk-epidemiologist"
```

Reply:
Really like this suggestion, thanks. Will do. 👍